source/class/cv/ConfigCache.js

```diff
@@ -87,36 +99,60 @@ qx.Class.define('cv.ConfigCache', {
         data: JSON.stringify(model.getWidgetDataModel()),
         addresses: model.getAddressList(),
         configSettings: JSON.stringify(cv.Config.configSettings),
-        config: cv.Config.configSuffix === null ? 'NULL' : cv.Config.configSuffix,
+        config:
+          cv.Config.configSuffix === null ? 'NULL' : cv.Config.configSuffix,
         body: document.querySelector('body').innerHTML
       });
     },
 
-    restore: function() {
+    restore() {
       const body = document.querySelector('body');
       const model = cv.data.Model.getInstance();
       this.getData().then(cache => {
         cv.Config.configSettings = cache.configSettings;
 
         // restore icons
-        cv.Config.configSettings.iconsFromConfig.forEach(function(icon) {
-          cv.IconHandler.getInstance().insert(icon.name, icon.uri, icon.type, icon.flavour, icon.color, icon.styling, icon.dynamic, icon.source);
+        cv.Config.configSettings.iconsFromConfig.forEach(function (icon) {
+          cv.IconHandler.getInstance().insert(
+            icon.name,
+            icon.uri,
+            icon.type,
+            icon.flavour,
+            icon.color,
+            icon.styling,
+            icon.dynamic,
+            icon.source
+          );
         }, this);
 
         // restore mappings
         if (cv.Config.configSettings.mappings) {
-          Object.keys(cv.Config.configSettings.mappings).forEach(function (name) {
+          Object.keys(cv.Config.configSettings.mappings).forEach(function (
+            name
+          ) {
             const mapping = cv.Config.configSettings.mappings[name];
             if (mapping && mapping.formulaSource) {
-              mapping.formula = new Function('x', 'var y;' + mapping.formulaSource + '; return y;'); // jshint ignore:line
+              mapping.formula = new Function(
+                'x',
+                'var y;' + mapping.formulaSource + '; return y;'
+              );
+              // jshint ignore:line
```

Comment:
Does that work - or does that comment need to be in the line above?

Reply:
As we do not use jshint anymore (replaced by eslint) this comment is obsolete anyways.